CHANGELOG.md

```diff
@@ -5,7 +5,14 @@ All notable changes to the ordercloud-javascript-sdk will be documented in this
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [4.6.6] - 20220-04-18
+
+## [4.6.7] - 2022-04-27
+### Fixed
+- bug where decoded token did not account for singular returned role
```

Comment:
Better to say "DecodedToken.role was not properly typed. It is now `(ApiRole[] | ApiRole | undefined)`